.env.example

```
@@ -157,6 +162,8 @@ BACKEND_FRONTEND_URL=http://localhost:5173
 # GOPRO_OVERLAY_BUILD_CONTEXT is copied into the image at /app/gopro-overlay.
 GOPRO_OVERLAY_BUILD_CONTEXT=https://github.com/capic2/gopro-dashboard-overlay.git#main
 GOPRO_OVERLAY_DATA_HOST_DIR=/media/nas/DS211_Synology_2/parapente
+# Persistent writable directory containing Codex auth.json. Treat it as a secret.
+CODEX_HOME_HOST_DIR=/home/capic/docker-data/dashboard-parapente/codex
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The CODEX_HOME_HOST_DIR key should go before the GOPRO_OVERLAY_BUILD_CONTEXT key

(UnorderedKey)